ush/generate_FV3LAM_wflow.sh

```diff
@@ -603,6 +603,19 @@ workflow launch script (WFLOW_LAUNCH_SCRIPT_FP):
   EXPTDIR = \"${EXPTDIR}\"
   WFLOW_LAUNCH_SCRIPT_FP = \"${WFLOW_LAUNCH_SCRIPT_FP}\""
 ln -fs "${WFLOW_LAUNCH_SCRIPT_FP}" "$EXPTDIR"
+
+print_info_msg "Generating an alternate simple launch script
+${EXPTDIR}/run_rocoto.sh"
+
+echo "#!/bin/bash" > ${EXPTDIR}/run_rocoto.sh
+if [[ $MACHINE == "wcoss2" ]] ; then
```

Comment:
@guoqing-noaa, it should be uppercases: wcoss2 => WCOSS2

Reply:
@chan-hoo Thanks a lot for catching this.  
I changed it to:
`
if [[ "${MACHINE,,}" == "wcoss2" ]] ; then
`
Hope it is a safer way and you are okay with it.